common/config/rush/nonbrowser-approved-packages.json

```diff
@@ -714,6 +714,10 @@
       "name": "node-forge",
       "allowedCategories": [ "libraries" ]
     },
+    {
+      "name": "node-object-hash",
```

Comment:
@D4N14L @dmichon-msft Any thoughts about how [node-object-hash](https://www.npmjs.com/package/node-object-hash) compares with Rush's existing methods for hashes (e.g. **repo-state.json**)?

Reply:
I did not see a ready to use utility that can hash object directly.
But, yeah, we can call Node.js `crypto.createHash` API to write a function to do that if we don't want to bring in another dependency

Reply:
Ok, I removed the `node-object-hash` dependency